source/geometry_model/chunk.cc

```diff
@@ -118,6 +166,17 @@ namespace aspect
       point1_lon = p1_lon;
     }
 
+#if !DEAL_II_VERSION_GTE(9,0,0)
+    template <int dim>
+    void
```

Comment:
you're going to get linker errors for older deal.II versions because you declare the function but then don't define it. please move the `#if` to only comment out the *body* of the function.